examples/nats-streaming-client/subscriber.bal

```diff
@@ -0,0 +1,26 @@
+import ballerina/encoding;
+import ballerina/io;
+import ballerina/log;
+import ballerina/nats;
+
+// Creates a NATS connection.
+nats:Connection conn = new("localhost:4222");
+
+// Initializes the NATS Streaming listener.
+listener nats:StreamingListener lis = new(conn);
+
+// Binds the consumer to listen to the messages published to the 'demo' subject.
+@nats:StreamingSubscriptionConfig {
+    subject: "demo"
+}
+service demoService on lis {
+    resource function onMessage(nats:StreamingMessage message) {
+       // Prints the incoming message in the console.
+       io:println("Message Received: " + encoding:byteArrayToString(message.getData()));
```

Comment:
Why are we using both `io` and `log` module functions in the same service? Shall we use only the `log` modules as this is a service